book/version_control.md

```diff
@@ -360,30 +292,39 @@ We can then define our cleaning instructions in a `.gitconfig` configuration fil
 
 ```
 git config --local include.path ../.gitconfig
 ```
 
-This configuration will now run the notebook output cleaning command every time a change to an `.ipynb` file is added to your local repo.
-You will need to then `git add` the file a second time to add the changes from the cleaning.
-This should prevent notebook outputs from being included in commits to this repository.
+This configuration will now run the notebook output cleaning command every time a change to an `.ipynb` file is added to your local repo. You will need to then `git add` the file a second time to add the changes from the cleaning. This should prevent notebook outputs from being included in commits to this repository.
 
 If you adjust the filter at any point, you should run this command to apply the changes to existing files:
 
 ```
 git add --renormalize .
 ```
 
-This approach has been borrowed from [this blog post by Yury Zhauniarovich](https://zhauniarovich.com/post/2020/2020-10-clearing-jupyter-output-p3/).
+This approach has been borrowed from this [blog post by Yury Zhauniarovich](https://zhauniarovich.com/post/2020/2020-10-clearing-jupyter-output-p3/).
+
+
+### Access control
+
+Remote version control platforms, like GitHub, have a [visibility setting](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/setting-repository-visibility) that represents whether the repo can be viewed publicly or only by owners of the project. We find that some analytical teams choose to develop code in the open, as per guidance from the [Government Service Manual](https://www.gov.uk/service-manual/service-standard/point-12-make-new-source-code-open). Others work in a private repositories, and then might either change to public or migrate to a fresh public repository to make the code open.
+
+```{caution}
+Access control should not be used instead of the good practices outlined above. Restricting access can help to limit the risk of releasing sensitive information, but correct use of version control tools avoids these issues.
+```
 
+On GitHub, Organizations provide an area for collating multiple repos that are associated with a particular team or department. Within Organizations, Teams can be also created to manage view and contribution permissions for specific projects within the Organization. External collaborators can also be added to projects, to allow direct contribution from those outside of the Organization. Detailed setup and management of Organizations and Teams are described in the [relevant section of the GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams).
 
-### Handling data breaches via Git repositories
+Regardless of the visibility status of a repo, only Organization members and collaborators may make direct contributions to the code in the repo. Others can contribute by Forking and using Pull Requests.
```

Comment:
What are the best practices with regards to reviews and merging. If I'm working on a repo I can set the repo to require merges to master only if a review has been done to the PR and the CI has passed. These settings are key for a good flow where no changes acommited without a review and CI tests. This is worth mentioning somewhere as part of maybe - Best Practices and recommendations of Github Setup or something just to outline to people the things they github page has to have. Not only do I mean like settings, but also descriptions, labels, license files, contributing guidelines. Sorry if this is covered below. I'm reading sequentially